docs/explanation/security.md

```diff
@@ -0,0 +1,63 @@
+(security)=
+
+# Security
+
+## Cryptographic technology
+
+Ops uses almost no cryptographic technology. Communication with Juju is done via reading environment variables ({external+juju:ref}`Hook <hook>`) and running processes in the charm environment ({external+juju:ref}`Hook command <list-of-hook-commands>`). There is no use of hashing or digital signatures.
+
+The one case where Ops uses cryptography is for sending trace data, when a certificates provider has been integrated with the charm. This is done using HTTPS, using either TLS 1.2 or 1.3, using the implementation provided by the standard library of the Python that is executing the charm. See [tracing - security considerations](#ops_tracing_security) for more details.
```

Comment:
Wow, does a hash-ref work across files? Sweet!

Reply:
When Sphinx processes the file, but not in GitHub, sadly.